### DISCUSSION TITLE
Due to Snap sandboxing, you need script. But I cannot find it.

### DISCUSSION THREAD

**Discussion by arxeiss** [keepassxreboot/keepassxc#9314] [Q&A] [answered]
Hello,

I'm bit confused with KeePassXC and the browser connection. I installed KeePassXC as **snap** and this is what I see in the settings.

1. Browser installed as snaps are not supported - well I have Firefox as Snap. But I'm using Chrome which I installed with `apt`. No issue here
2. Due to Snap sandboxing, you must run script. I installed KeePassXC as snap, so I might need to run that script. But I navigated to the provided link and I don't see any script on the KeePassXC homepage, nor Docs/FAQ tab nor https://keepassxc.org/docs/KeePassXC_UserGuide etc.

I was searching in Github issues and here and I cannot find solution. Is the solution to remove KeePassXC from snap and add your PPA?

![image](https://user-images.githubusercontent.com/5103109/231412840-e4313e01-81c6-40dd-b1f2-7cba6651fe25.png)

**Comment by phoerious** [comment, marked as answer]
We forgot to add the link to the script after the website relaunch. It's there now.
